libavcodec/vvc/sei.h

```diff
@@ -41,9 +41,18 @@ typedef struct VVCSEIPictureHash {
     uint8_t hash_type;
 } VVCSEIPictureHash;
 
+typedef struct VVCSEIFrameFieldInfo {
+    int present;
+    int picture_struct;
+    uint8_t display_elemental_periods;
+    uint8_t source_scan_type;
+    uint8_t duplicate_flag;
+} VVCSEIFrameFieldInfo;
+
 typedef struct VVCSEI {
 	H2645SEI common;
     VVCSEIPictureHash picture_hash;
+    VVCSEIFrameFieldInfo frame_field_info;
```

Comment:
this can be defined in h274.h too, right?

Reply:
Sure. Will update VVCSET  to H274SEI and move to h274.h